articles/multifactor-authentication/api/index.md

```diff
@@ -1,69 +1,53 @@
+Before you can use the MFA APIs, you'll need to:
+
+* Enable the MFA grant type for your application. You can enable the MFA grant by going to [Applications > Your Application > Advanced Settings > Grant Types](${manage_url}/#/applications) and selecting MFA.
+* Create a rule that sets Guardian as the MFA provider. For more information, see [Guardian for Administrators](/multifactor-authentication/administrator).
 
 If you are using the MFA API in conjunction with a [Token endpoint](/api/authentication#get-token), you must meet the requirements of the corresponding grant.
 
-## Enforcing MFA when interacting with the Token Endpoint
+## Multifactor authentication with the Token endpoint
 
 We have expanded MFA support on the Token endpoints to cover the following use cases:
 
-* Use MFA with the [password](/api-auth/grant/password), [password-realm](/api-auth/grant/password#realm-support), [refresh-token](/tokens/refresh-token/current#use-a-refresh-token) grants
-* Completion of first-time enrollment by end-users during authentication
-* Selection of the desired MFA authenticator by the end-user before they execute the MFA challenge
-<!-- * Support for TOPT delivered via Email -->
-<!-- TODO: Add link to (to be created) doc about email authenticator -->
+* Use MFA with the [password](/api-auth/grant/password), [password-realm](/api-auth/grant/password#realm-support), and [refresh-token](/tokens/refresh-token/current#use-a-refresh-token) grants.
+* Completion of first-time enrollment by users during authentication.
+* Selection of the desired MFA authenticator by the user before they execute the MFA challenge.
 
 ### More info
 
 ::: next-steps
 * [Manually triggering MFA challenges](/multifactor-authentication/api/challenges)
 * [Using one-time passwords as the MFA challenge](/multifactor-authentication/api/otp)
 * [Using SMS messages as the MFA challenge](/multifactor-authentication/api/oob)
-<!-- * [Using Push Notifications]
-* [Using one-time passwords via email] -->
-<!-- TODO: Add missing articles and link to them -->
 * [Tutorial: How to use MFA with the Resource Owner Password Grant](/api-auth/tutorials/multifactor-resource-owner-password)
 :::
 
-## Programmatic enrollment and management of user authenticators
-
-The MFA Associate API allows you to create, read, update, and delete authenticators.
-
-You can use this API to power your end-user interfaces where your users can:
-
-* Manage their MFA enrollments
-* Add new MFA authenticators
-* Remove existing MFA authenticators
+## Enrollment and management of user authenticators
 
-This is useful if you want your users to be able to:
+The MFA Associate API allows you to create, read, update, and delete authenticators. You can use this API to power user interfaces where users can manage MFA enrollments, or add and remove authenticators.
```

Comment:
sounds like this is a different API, while it's 3 different endpoints that are all part of the Authentication API (I would go as far as to ask why, since this feels like it should be part of the Mgmt API, but that's a product decision)